gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/use_case/ImportCRDUseCase.java

```diff
@@ -286,7 +286,7 @@ private ApiCRDStatus update(Input input, Api existingApi) {
                         return Map.entry(
                             key,
                             updatePlanDomainService
-                                .update(initPlanFromCRD(plan), plan.getFlows(), existingPlanStatuses, api, input.auditInfo)
+                                .updateV4ApiPlan(initPlanFromCRD(plan), plan.getFlows(), existingPlanStatuses, api, input.auditInfo)
```

Comment:
done, but I had to extend update() with a new parameter because from what I see sometimes we want to provide existing plans: 
<img width="1107" alt="image" src="https://github.com/user-attachments/assets/bdd3a58f-ca17-4aac-ac92-fc6774383a91">